src/Raven.Studio/typescript/models/database/index/testIndex.ts

```diff
@@ -58,6 +58,10 @@ class testIndex {
         this.dbProvider = dbProvider;
         this.indexDefinitionProvider = indexDefinitionProvider;
 
+        if (indexDefinitionProvider().name()) {
+            this.query(`from index "${indexDefinitionProvider().name()}"`)
```

Comment:
what's the purpose of that?

Reply:
In case of new index without provided name, this field equals "", and it stay in query as "<TestIndexName>".
If name is provided, then it is present in query.